10/umbraco-cms/extending/health-check/guides/macroerrors.md

```diff
@@ -5,7 +5,7 @@
 
 # Health check: Macro errors
 
-_Checks to make sure macro errors are not set to throw a YSOD (yellow screen of death), which would prevent certain or all pages from loading completely._
+_Checks to make sure macro errors are not set to throw a YSOD: Yellow Screen Of Death. This could prevent certain or all pages from loading._
```

Comment:
```suggestion
_Checks to make sure macro errors are not set to throw a Yellow Screen Of Death (YSOD). This could prevent certain or all pages from loading._
```

Reply:
So I changed this in order to test and verify the changes to the acronym rule, so it's valid enough 😅 
Any reason you want to change it back again?

Reply:
Ah, I see.. I changed it to maintain consistency with the other file where the same acronym is used 😀

But you can leave it as it is 💪

Reply:
Yeah, good point! Let's keep it consistent 💪